plugins/textpattern.md

```diff
@@ -101,10 +101,10 @@ The block patterns do not have an `end` property. This allows for patterns to be
 
 ```js
 tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
-  plugin: 'textpattern lists',
-  textpattern_patterns: [
-    {start: '#', format: 'h1'},
+  selector: 'textarea',          // => change this value according to your HTML
```

Comment:
Why did we add the `=>` here? This is just confusing to me.

Reply:
It was to specify the code line that it belongs too. Removing it...